README.md

```diff
@@ -37,22 +44,133 @@ api("com.github.spotify:openfeature-kotlin-sdk:[ANY_BRANCH]-SNAPSHOT")
 
 This will get a build from the head of the mentioned branch. 
 
-### Resolving a flag
+## 🌟 Features
+
+- support for various backend [providers](https://openfeature.dev/docs/reference/concepts/provider)
+- easy integration and extension via [hooks](https://openfeature.dev/docs/reference/concepts/hooks)
+- bool, string, numeric, and object flag types
+- [context-aware](https://openfeature.dev/docs/reference/concepts/evaluation-context) evaluation
+
+## 🚀 Usage
+
 ```kotlin
-import dev.openfeature.sdk.*
+    // configure a provider and get client
+    OpenFeatureAPI.setProvider(
+        CustomProvider.initialise()
```

Comment:
"initialize()" is called automatically by "setProvider". I'd remove this from here and just pass "customProvider" as argument